src/peft/utils/other.py

```diff
@@ -460,6 +471,7 @@ def get_auto_gptq_quant_linear(gptq_quantization_config):
 
 TRANSFORMERS_MODELS_TO_PREFIX_TUNING_POSTPROCESS_MAPPING = {
     "bloom": bloom_model_postprocess_past_key_value,
+    "gpt_bigcode": starcoder_model_postprocess_past_key_value,
```

Comment:
Can we be sure that this model type will only match starcoder and not, for instance, santacoder? Or is the post-processing the same?

Reply:
Hello, `santacoder` is just a smaller version, i.e.,  post-processing is the same